releasenotes/notes/fixproblemswithaltwindir-afc7228d0f135400.yaml

```diff
@@ -12,3 +12,11 @@ fixes:
     problem where `logs` was effectively hardcoded to use `c:` for programdata
     Fixes installation problem where process & trace service were using
     `c:\programdata\...` to find datadog.yaml regardless of installation dir
+
+    If upgrading from a prior version, the configuration file (datadog.yaml) may 
+    have incorrect data.  It will be necessary to manually update those entries.
+    For example
+    `confd_path: c:\programdata\datadog\conf.d`
+    will have to be changed to
+    `confd_path: d:\programdata\datadog\conf.d`
```

Comment:
nit: the release notes are in reST format, not markdown, so inline code needs to be formatted with double backticks instead of single backticks. Can you fix the whole release note accordingly?